assets/src/util/mapMode.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import { ReactElement } from "react"
+import MapPage from "../components/mapPage"
+import SearchPage from "../components/searchPage"
+import inTestGroup, { MAP_BETA_GROUP_NAME } from "../userTestGroups"
+
+export interface NavMode {
+  path: string
+  title: string
+  element: ReactElement
+}
+
+export const mapModeForUser = (): any =>
```

Comment:
question: should this return `any` if there's the `NavMode` interface above?

Reply:
💯, thanks!